README.md

```diff
@@ -237,6 +237,37 @@ a single line.
 Prefer writing pure functions, which means they do not manipulate the input arguments or
 reference/manipulate global state. This makes your code better scalable and testable.
 
+#### Arrow functions
```

Comment:
Maybe we should add
```
### Functions
```
above the line `#### Pure functions` to group both chapters together

Reply:
I added a separate section for functions and made the rest of the sections in the same level!